### PR TITLE
[python] fixed sklearn test on python 2.7

### PR DESCRIPTION
**Pull request by StrikerRUS**
Refer to https://github.com/Microsoft/LightGBM/pull/880#issuecomment-326837486

### CODE REVIEW

**Comment by wxchan** [comment]
Actually I don't think we need to check num of deprecated warnings.

**Comment by StrikerRUS** [comment]
@wxchan Remain only check that `seed` and `random_state` internally are the same and remove check for "warnings were raised"?

**Comment by wxchan** [comment]
@StrikerRUS yes, what do you think?

**Comment by StrikerRUS** [comment]
@wxchan I agree. Anyway, in the future warnings will be deleted and "warnings were raised" check correspondingly too.
